<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HydroCoupleVisClass</class>
 <widget class="QMainWindow" name="HydroCoupleVisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HydroCoupleVis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GraphicsView" name="graphicsViewHydroCoupleVis" native="true">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBarMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1486</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="toolTip">
     <string>File</string>
    </property>
    <property name="statusTip">
     <string>File</string>
    </property>
    <property name="whatsThis">
     <string>File</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
    <widget class="QToolBar" name="toolBarMain">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="toolTip">
    <string>Main Toolbar</string>
   </property>
   <property name="statusTip">
    <string>Main Toolbar</string>
   </property>
   <property name="whatsThis">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionDefaultSelect"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomExtent"/>
   <addaction name="actionPan"/>
  </widget>
  <widget class="QStatusBar" name="statusbarMain"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Model Components</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeView" name="treeViewModelComponents"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/new</normaloff>:/HydroCoupleVis/new</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>New Project</string>
   </property>
   <property name="whatsThis">
    <string>Creates a new project.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/open</normaloff>:/HydroCoupleVis/open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open File</string>
   </property>
   <property name="whatsThis">
    <string>Opens a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/save</normaloff>:/HydroCoupleVis/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save </string>
   </property>
   <property name="whatsThis">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/saveas</normaloff>:/HydroCoupleVis/saveas</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="statusTip">
    <string>Save As</string>
   </property>
   <property name="whatsThis">
    <string>Save as.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/print</normaloff>:/HydroCoupleVis/print</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="statusTip">
    <string>Print</string>
   </property>
   <property name="whatsThis">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/close</normaloff>:/HydroCoupleVis/close</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close</string>
   </property>
   <property name="whatsThis">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionDefaultSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/default</normaloff>:/HydroCoupleVis/default</iconset>
   </property>
   <property name="text">
    <string>actionDefaultSelect</string>
   </property>
   <property name="toolTip">
    <string> Select</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/zoomin</normaloff>:/HydroCoupleVis/zoomin</iconset>
   </property>
   <property name="text">
    <string>actionZoomIn</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/zoomout</normaloff>:/HydroCoupleVis/zoomout</iconset>
   </property>
   <property name="text">
    <string>actionZoomOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoomExtent">
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/zoomextent</normaloff>:/HydroCoupleVis/zoomextent</iconset>
   </property>
   <property name="text">
    <string>actionZoomExtent</string>
   </property>
   <property name="toolTip">
    <string>Zoom Extent</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/hydrocouplevis.qrc">
     <normaloff>:/HydroCoupleVis/pan</normaloff>:/HydroCoupleVis/pan</iconset>
   </property>
   <property name="text">
    <string>actionPan</string>
   </property>
   <property name="toolTip">
    <string>Pan</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About HydroCouple Composer</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QWidget</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
